cypress/pages/AddEmployeePage.ts

```diff
@@ -0,0 +1,34 @@
+import BasePage from "./BasePage";
+
+class AddEmployeePage extends BasePage {
+    // locators
+    firstNameInput: string = "[name='firstName']";
+    middleNameInput: string = "[name='middleName']";
+    lastNameInput: string = "[name='lastName']";
+    createLoginDetailsToggle: string = ".oxd-switch-input";
+    usernameInput: string = ":nth-child(4) > .oxd-grid-2 > :nth-child(1) > .oxd-input-group > :nth-child(2) > .oxd-input";
+    passwordInput: string = ".user-password-cell > .oxd-input-group > :nth-child(2) > .oxd-input";
+    confirmPasswordInput: string = ".oxd-grid-2 > :nth-child(2) > .oxd-input-group > :nth-child(2) > .oxd-input";
+    AddProfileImageButton: string = ".oxd-file-div .oxd-icon-button"
+    fileInput: string = "[type='file']";
+    saveButton: string = ".oxd-form-actions [type='submit']";
+    employeeIdInput: string = ".oxd-grid-2 .oxd-grid-item input";
+
+    // variables
+    profileImagePath: string = "/home/yazan/Desktop/cypress-app/assets/images/profilePhoto.jpg";
```

Comment:
we should not have var in page object

Reply:
it is a variable related to the page object, and to prevent magic string